Dockerfile-dev

```diff
@@ -1,4 +1,4 @@
-FROM node:12-alpine
+FROM node:14-alpine
```

Comment:
might also need to update: https://github.com/CSCfi/metadata-submitter/blob/develop/Dockerfile#L1

Reply:
true